src/Button/index.js

```diff
@@ -311,44 +343,48 @@ const Button = (props: Props) => {
 
   return (
     <StyledButton
+      {...props}
```

Comment:
why this order ?

Reply:
Because I wanted to prevent overriding of `disabled` prop by `{...props}` object.

Reply:
Sorry, I don't understand. disabled is in props, so what do you mean by prevent overriding of `disabled` ?


![image](https://user-images.githubusercontent.com/7850592/45164022-db57d000-b1f1-11e8-88a4-6d52d0f43759.png)

Reply:
Ohh, i see. `disabled` is calculated in component, Is it really necessary passing other props to button ? 🤔